packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -79,6 +79,40 @@ export interface BundlingOptions {
    * @experimental
    */
   readonly local?: ILocalBundling;
+
+  /**
+   * The type of packaging to apply on the bundling output.
+   *
+   * @default BundlePackaging.AUTO
+   *
+   * @experimental
+   */
+  readonly packaging?: BundlePackaging;
+}
+
+/**
+ * The type of packaging to apply on the bundled asset.
+ *
+ * @experimental
+ */
+export enum BundlePackaging {
```

Comment:
Not sure the semantics are perfect because this is not about "packaging the bundle".

Perhaps we can consider the mental model "What type of output does the bundling process produce?", in which case I'd consider names like `BundlingOutput.ARCHIVED` and `BundlingOutput.NOT_ARCHIVED`

WDTY?

Reply:
Then it's a boolean?

Reply:
We also want `BuildingOutput.AUTO_DISCOVER`

Reply:
Just to be sure: `BundlingOutput.NOT_ARCHIVED` means *what I'm producing with my bundling is not archived, please archive it* and `BundlingOutput.ARCHIVED` *what I'm producing is an archive, please do not rearchive it*?

Reply:
Yes, names are not great, but I couldn't come up with something better.